lib/tools/adb-commands.js

```diff
@@ -361,7 +361,7 @@ methods.hiddenApiPolicyP = async function () {
  *
  * Commands don't lead crash or something, thus simply this method calls deleting all setting.
  */
-methods.defaultHiddenApiPolicyP = async function () {
+methods.setDefaultHiddenApiPolicy = async function () {
```

Comment:
how about `enableNonSdkApiAccess`/`disableNonSdkApiAccess`?

Reply:
I also consider so.  But the default preference probably depends on companies which customise the devices, I guess. Google defines as below. If we use _enable/disable_, putting _2 or 3_/_0 or 1_ is probably proper. But if the value isn't the default value for the device, it isn't suitable for Appium to reset the value. So, I chose `relax` and `setDefault`

```
0: Disable non-SDK API usage detection. This will also disable logging, and also break the strict mode API, detectNonSdkApiUsage(). Not recommended.
1: "Just warn" - permit access to all non-SDK APIs, but keep warnings in the log. The strict mode API will keep working.
2: Disallow usage of dark grey and black listed APIs.
3: Disallow usage of blacklisted APIs, but allow usage of dark grey listed APIs.
```

Do you think enable/disable are proper?

Reply:
Then I would rather have two methods for resettings the values to their defaults and for setting them to the particular constant from the list above. It might be also handy to mention the stuff above in the docstring